src/components/Messages/MessageIcon.tsx

```diff
@@ -0,0 +1,120 @@
+import useXmtpClient from '@components/utils/hooks/useXmtpClient';
```

Comment:
We might want to name this component better. MessageIcon component having the code for streaming messages is a bit out of context